src/main/kotlin/study/racingcar/domain/RacingCar.kt

```diff
@@ -0,0 +1,21 @@
+package study.racingcar.domain
+
+/**
+ * 기능
+ *    - 식별할 수 있는 id 와 위치값 상태를 갖는다.
+ *    - 앞뒤로 이동해서 위치값 상태를 수정하는 함수를 제공한다.
+ *    - 초기 위치보다 뒤로 이동하지는 않는다.
+ * */
+class RacingCar(
+    val id: String,
+    var position: Int
+) {
```

Comment:
position의 가시성이 public이라서 외부에서 맘대로 수정이 가능합니다.

변경은 객체 내에서만 행해질수 있도록 하고 읽기를 위한 기능을 제공해보면 좋을것 같습니다.

```suggestion
    private var position: Int
) {
    val currentPosition: Int
        get() = position
```